electron-builder.yml

```diff
@@ -3,6 +3,12 @@ appId: io.ipfs.desktop
 directories:
   buildResources: assets/build
 
+extraFiles:
```

Comment:
@lidel Probably we could use `extraResources` here, no? https://www.electron.build/configuration/contents#extraresources

Just to make sure it goes to `resources` on Linux and `Resources` on macOS.

Reply:
I've updated this on 7c5f6bba63c534f2e34f5d4f2b1fed2c031e7ad9. @lidel could you please make sure it still works on Linux?

Reply:
Tested, works fine :)